tests/profile_speed.py

```diff
@@ -133,9 +140,11 @@ def convert_size(size_bytes):
         'system': uname.system,
         'machine': uname.machine,
         'processor': uname.processor,
+        'n_cpu': multiprocessing.cpu_count(),
```

Comment:
Will this report the n_cpus available, or the actual n_cpus used? I think we want to keep track of the latter?

Reply:
This will report n_cpus available. Not number used. But when we parallelize the script, we should do it across all available CPUs so that they are the same. For number used, I'd just have to put 1 for now. Maybe I can add a column that says `n_workers`?

Reply:
yes we should add a column, `n_cpu` and `n_workers` sgtm